client/src/components/createLanguageRedirect.js

```diff
@@ -4,10 +4,12 @@ const createLanguageRedirect = ({ clientLocale, lang }) => {
 
   let path = window?.location?.pathname?.split('/');
   path = path
-    .filter(item => (item !== clientLocale && item !== lang ? item : ''))
+    .filter(
+      (item, i) => i !== 0 || (item !== clientLocale && item !== lang && item)
+    )
     .join('/');
 
-  const domain = window?.location?.host
+  const domain = window?.location?.hostname
```

Comment:
Any reason we would not want `host` (not that we need it, but is inclusive of port and can be used in future for tests, elsewhere?)

Reply:
I guess not. I just saw the name `domain` and thought that would mean we didn't want the port.

Maybe hostTail or some other slightly awkward, but informative, name would work better.